src/libraries/System.Net.Http.WinHttpHandler/ref/System.Net.Http.WinHttpHandler.cs

```diff
@@ -19,6 +19,10 @@ public enum WindowsProxyUsePolicy
         UseWinInetProxy = 2,
         UseCustomProxy = 3,
     }
+    public static partial class WinHttpExtensions
+    {
+        public static System.Net.Http.Headers.HttpHeaders GetWinHttpTrailingHeaders(this System.Net.Http.HttpResponseMessage responseMessage) { throw null; }
+    }
```

Comment:
In our offline discussions @geoffkizer and others pushed back against my explicit API idea. The team decision is that we want to deliberately reduce the accessibility and visibility of the .NET Framework hack.

This should be undone now, we are back to your original proposal as-is, and no API review is needed.